sdk/monitor/monitor-opentelemetry-exporter/test/common/scenario/basic.ts

```diff
@@ -37,41 +37,36 @@ export class BasicScenario implements Scenario {
         foo: "bar"
       }
     });
-    await opentelemetry.context.with(
-      opentelemetry.setSpan(opentelemetry.context.active(), root),
-      async () => {
-        const child1 = tracer.startSpan(`${this.constructor.name}.Child.1`, {
-          startTime: 0,
-          kind: opentelemetry.SpanKind.CLIENT,
-          attributes: {
-            numbers: "123"
-          }
-        });
-
-        const child2 = tracer.startSpan(`${this.constructor.name}.Child.2`, {
-          startTime: 0,
-          kind: opentelemetry.SpanKind.CLIENT,
-          attributes: {
-            numbers: "1234"
-          }
-        });
-
-        opentelemetry.context.with(
-          opentelemetry.setSpan(opentelemetry.context.active(), child1),
-          () => {
-            child1.setStatus({ code: SpanStatusCode.OK });
-            child1.end(100);
-          }
-        );
-
-        await delay(0);
-        child2.setStatus({ code: SpanStatusCode.OK });
-        child2.end(100);
-
-        root.setStatus({ code: SpanStatusCode.OK });
-        root.end(600);
-      }
+    const ctx = opentelemetry.setSpan(opentelemetry.context.active(), root);
```

Comment:
Dumb question perhaps - isn't the `context.with()` supposed to change the `context.active()` automatically? Ie shouldn't the tracer.startSpan calls within the 'with' been parented properly?

Or is this fix different?

Reply:
From the exporter perspective we were not getting parentId in this test so I went and took a look at current sample code in OT, https://github.com/open-telemetry/opentelemetry-js/blob/main/examples/basic-tracer-node/index.js and context need to be added as a parameter now, I'm not sure what are all the possible approaches here., @MSNev is this something you can help?

Reply:
While I'm nowhere near an expert on this, based on the test is doing and the sample... I think this is ok.